src/components/LessonCard.tsx

```diff
@@ -96,7 +101,7 @@ const LessonCard: React.FC<LessonCardsProps> = (props) => {
         <div className="card__info">
           <div className="card__info-top">
             <Title title={props.title} />
-            <MenuKebab />
+            <MenuKebab idCard={props.id}/>
```

Comment:
And for pretty view you need a space before "/"
... idCard={props.id} /...

Reply:
Done